src/open_inwoner/openklant/services.py

```diff
@@ -1372,6 +1459,55 @@ def create_question(
         )
         logger.info("Created taak: %s", taak["uuid"])
 
+        return taak
+
+    def create_question_for_partij(
+        self,
+        partij_uuid: str,
+        question: str,
+        subject: str,
+    ) -> OpenKlant2Question:
+        klantcontact = self._create_klantcontact(question=question, subject=subject)
+        self._create_betrokkene_in_klantcontact(
+            klantcontact_uuid=klantcontact["uuid"],
+            betrokkene_data={"wasPartij": {"uuid": partij_uuid}},
+        )
+        self._create_interne_taak(klantcontact_uuid=klantcontact["uuid"])
+
+        return OpenKlant2Question.from_klantcontact_and_answer(klantcontact)
+
+    def create_question_for_betrokkene(
```

Comment:
Nitpick, but: you are  creating a question _and_ a betrokkene here. Perhaps we should reflect that in the name, e.g.:

```suggestion
    def create_question_with_betrokkene(
```